django_mongodb_cli/config.py

```diff
@@ -0,0 +1,261 @@
+from os.path import join
+
+test_settings_map = {
```

Comment:
The restructuring would help reduce some of the directory jumping you've got to do here. Again follow-up for when we have to expand this beyond the starter 5.